spec/support/features/in_person_helper.rb

```diff
@@ -202,4 +190,41 @@ def mark_in_person_enrollment_passed(user)
     enrollment.profile.activate_after_passing_in_person
     enrollment.update(status: :passed)
   end
+
+  def perform_mobile_hybrid_steps
```

Comment:
The inclusion of these steps in the diff suggests that there was an incorrectly resolved merge conflict.